src/main/java/org/folio/services/validator/InvoiceValidator.java

```diff
@@ -98,7 +105,7 @@ public void validateBeforeApproval(Invoice invoice, List<InvoiceLine> lines) {
     validateInvoiceTotals(invoice);
     verifyInvoiceLineNotEmpty(lines);
     validateInvoiceLineFundDistributions(lines, Monetary.getCurrency(invoice.getCurrency()));
-    validateInvoiceAdjustmentsDistributions(adjustmentsService.getNotProratedAdjustments(invoice) , Monetary.getCurrency(invoice.getCurrency()));
+    validateInvoiceAdjustmentsDistributions(adjustmentsService.getNotProratedAdjustments(invoice),Monetary.getCurrency(invoice.getCurrency()));
```

Comment:
I see couple of places with incorrect formatting of commas, please format it. The correct formatting: comma + space + next operand

Reply:
done